mass_editing/migrations/10.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def migrate(cr, version):
+    if not version:
+        return
+
+    # Check if table has already migrated
+    try:
+        cr.execute(
```

Comment:
I think that this line will always fail, as pre-migration script has renamed mass_object. 
Maybe move the post-migration in the pre-migration will fix the problem. 
You should only add a script that add a table. Don't you think ?

Otherwise, the concept is correct. Thansk !

Reply:
About your cue, I got idea from server-ux repository, branch 13.0. There are pre-migration.py and post-migration.py as this, and I kept this structure for readability reasons.
Guess my last modify is tricky, I want to test your suggestion.

Thank you.

Reply:
Indeed, the 13.0 was the only one migration script. Now if, this migration script is copied in many versions, it is not valid anymore and should be adapted.